internal/cmd/web.go

```diff
@@ -22,9 +24,33 @@ var WebCMD = cli.Command{
 			Usage: "Artifact directory",
 			Value: "/tmp/artifacts",
 		},
+		&cli.StringFlag{
+			Name:  "builds-dir",
+			Usage: "Directory to store build jobs and their artifacts",
+			Value: "/tmp/kairos-builds",
+		},
+		&cli.BoolFlag{
+			Name:  "create-worker",
+			Usage: "Start a local worker in a goroutine",
+			Value: false,
+		},
 	},
 	Action: func(c *cli.Context) error {
 		os.MkdirAll(c.String("artifact-dir"), os.ModePerm)
-		return web.App(c.String("address"), c.String("artifact-dir"))
+		os.MkdirAll(c.String("builds-dir"), os.ModePerm)
+
+		// If create-worker flag is set, start a worker in a goroutine
+		if c.Bool("create-worker") {
+			workerID := "local-worker"
+			w := worker.NewWorker("http://localhost"+c.String("address"), workerID)
```

Comment:
can we rename address to port?

Reply:
I don't mind changing it. But it comes from the command's `Flags` above and it was called `address` so far. Since we don't use the web interface in CI or anything like that, I don't expect anything to break from this change.

Reply:
if we don't have much deps on this yet I'd say let's do but don't let that block merging

Reply:
Actually, the "address" field, is propagated down to: https://go.dev/src/net/http/server.go#L2959

which is indeed the full address. It is something like `host:port` with `:port` also being valid. I think the right fix is to change the code here, split in `:` and take the second part (the port).

Reply:
Fixed.